synapse/storage/schema/delta/55/add_spans_to_device_lists.sql

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 The Matrix.org Foundation C.I.C
```

Comment:
this will need to be in `delta/56`, otherwise it won't get run on synapses which have already upgraded to v56.